lib/mail_room/cli.rb

```diff
@@ -42,10 +42,12 @@ def initialize(args)
     # Start the coordinator running, sets up signal traps
     def start
       Signal.trap(:INT) do
+        structured_logger.info(action: "Interrupt signal received")
         coordinator.running = false
       end
 
       Signal.trap(:TERM) do
+        structured_logger.info(action: "Terminate signal received")
```

Comment:
I'm not sure this is necessary, as the process will have exited without an error message being logged. I may be wrong, open to hearing thoughts here.

Reply:
I'm not really sure myself. If there are no strong opinions (I see pros and cons - possibly useful information left out vs some possibly useless code). I don't have a strong opinion, but I'd probably err to leaving it in 🤔

Reply:
I'd say leave this one out because it'll be tough to get access without the global logger.

Reply:
Sounds good, I'll remove it 👍